runbotics/runbotics-orchestrator-ui/src/src-app/components/Tile/BotCollectionTile/BotCollectionTile.actions.tsx

```diff
@@ -15,13 +14,14 @@ import BotCollectionModifyOption from '#src-app/views/bot/BotCollectionView/Dial
 
 import { BotCollectionTileProps } from './BotCollectionTile.types';
 
-
 const BotCollectionTileAction: VFC<BotCollectionTileProps> = ({ botCollection, displayMode }) => {
     const [anchorEl, setAnchorEl] = React.useState<HTMLElement>(null);
     const { translate } = useTranslations();
     const hasDeleteBotCollectionAccess = useFeatureKey([FeatureKey.BOT_COLLECTION_DELETE]);
     const hasEditBotCollectionAccess = useFeatureKey([FeatureKey.BOT_COLLECTION_EDIT]);
 
+    const isDeleteAndModifyVisible = !['Guest', 'Public'].includes(botCollection.name);
```

Comment:
create enum in `runbotics-common` that will provide such values

Reply:
just fyi if its constant that does not depend on any prop or state then declare it outside of the component